### ISSUE TITLE
CMS Editors Should Not Be Able to Rename/Re-URL-alias VAMCs Facility Services 

### ISSUE THREAD

**Issue by kmariepat-cityfriends** [department-of-veterans-affairs/va.gov-cms#14136]
VAMC facility services once created, cannot be changed to a different type of service (which would ultimately change the title and the URL). 

Note, we will not do this for VBA immediately to ensure cloning is allowed to stand up facilities 

##### Acceptance Criteria 
- [x] I expect that editors should not be able to change the title, service name, and/or URL of existing VAMC facility service nodes
- [x] ~~I expect Administrators can change the title, service name, and/or URL of existing Vet Center AND VAMC facility service nodes (this would be carried out request fulfillment)~~ Not in scope
- [x] I expect this follows the same model as used for Vet Centers (see #11367)
- [x] Cypress tests have been added that verify editors are unable to change the title, service name, and/or URL of existing Vet Center AND VAMC facility service nodes
- [x] Cypress tests have been added that verify Administrators can change the title, service name, and/or URL of exisiting Vet Center AND VAMC facility service nodes
- [x] I expect that VAMC System Health services are reviewed to see if a similar issue exists and a separate ticket is stubbed out if so

**Comment by kmariepat-cityfriends** [comment]
Approach for this ticket, aligns to where we land here: https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/4915

**Comment by xiongjaneg** [comment]
Noted in 16th minute that Administrator only applies to full Admin, not Content Admin.

**Comment by Becapa** [comment]
VAMC System Health Service nodes also need this treatment to prevent editors from changing the service. Ticket created: #17349

**Comment by jilladams** [comment]
Same Q here: PR merged to main, and I think this should deploy when CMS CI issues are unblocked, so I'm gonna bump it to "Pending prod verification" but let me know if I'm missing something.

**Comment by jilladams** [comment]
Verifying the changes in prod:
- 🟢  I expect that editors should not be able to change the title, service name, and/or URL of existing VAMC facility service nodes
- 🟢   I expect Administrators can change the title, service name of existing Vet Center AND VAMC facility service nodes (this would be carried out request fulfillment)
- 🔴   I expect Administrators can change URL of existing Vet Center AND VAMC facility service nodes (this would be carried out request fulfillment) -- https://staging.cms.va.gov/node/66355/edit, as Admin, the URL alias field doesn't appear. Not sure if that's a change from before or always been hidden for admins?

On Vet Center ones you can add a URL Redirect, but not on VAMC Facility ones.

**Comment by xiongjaneg** [comment]
@Becapa Can you check Jill's prior comment about verifying Admin URL change and add a response? Thanks!

**Comment by Becapa** [comment]
Oh, that wasn't something that I changed, so I think it was always hidden. My thought with this ticket was that by changing the title and service name of the VAMC facility service it ultimately also changes the URL and that this functionality, in particular, was what we were trying to prevent for editors, but allow for admins. 
The URL Redirect field is hidden at the node edit form level via config. So, I think whether it is hidden or not is the same for everyone who can access the form. I imagine we could use an alter hook to change the rendering of the form if we need to based on permissions, though.

**Comment by xiongjaneg** [comment]
I crossed off the Admin related AC as that seems out of scope due to adding new functionality that isn't currently there. That should be considered on its own, if needed. That AC was not on the Vet Center ticket that was referenced, so crossing this off aligns with that AC. Other ACs are confirmed.
